Src/FluentAssertions/NumericAssertionsExtensions.cs

```diff
@@ -9,6 +9,8 @@ namespace FluentAssertions
     /// </summary>
     public static class NumericAssertionsExtensions
     {
+
+        #region BeCloseTo
```

Comment:
🔧 I've become more fond of using partial classes instead of `#region`s for cases such as this.
We could extract this into four partial classes `NumericAssertionsExtensions.BeCloseTo.cs` etc.

Reply:
I agree that partial classes would be even more maintainable.
@dennisdoomen since such an approach is not yet implemented anywhere in the codebase, could you comment on that?

Reply:
You don't even need `partial` classes here.

Reply:
Do you mean simply defining few classes, one for each method group(`BeCloseNumericAssertionExtension`, `NotBeCloseNumericAssertionExtensions` and so on)?

Reply:
Yes, you could do that. Although I personally prefer regions for this. Being able to quickly find all the assertions from Github is something I often do.

Reply:
Let's stick with regions for now then.